server/src/test/java/org/elasticsearch/index/mapper/BinaryFieldMapperTests.java

```diff
@@ -255,7 +255,7 @@ private void mapping(XContentBuilder b) throws IOException {
                 b.field("type", "binary").field("doc_values", "true");
 
                 if (rarely()) {
-                    b.field("store", true);
+                    b.field("store", false);
```

Comment:
can this if statement then completely be removed? Given that `store` defaults to `false`?

Reply:
Yes, but this is what we do in other places so i figured it won't hurt.